lib_test/test_rewrite.ml

```diff
@@ -98,8 +101,8 @@ module Constructors = struct
       add_redirect t ip_packet
           ((V4 internal_xl), internal_xl_port)
           ((V4 internal_client), internal_client_port) >>= function
-      | Ok -> Lwt.return t
-      | Overlap | Unparseable -> Alcotest.fail "Failed to insert test data into table structure"
+      | Ok () -> Lwt.return t
+      | Error _ -> Alcotest.fail "Failed to insert test data into table structure"
```

Comment:
Why throw out the error here? We almost certainly want it if we've triggered a test failure.